app/src/main/java/com/innowisegroup/datetimepicker/DateTimePickerDialog.kt

```diff
@@ -180,7 +207,42 @@ class DateTimePickerDialog(
         this.show(fragmentManager, DIALOG_TAG)
     }
 
-    private companion object {
-        const val DIALOG_TAG = "date_time_picker_dialog"
+    companion object {
+        const val UPDATE_TIME_TAB_TITLE_REQUEST_KEY = "updateTimeTabTitleKey"
+        const val UPDATE_DATE_TAB_TITLE_REQUEST_KEY = "updateDateTabTitleKey"
+
+        const val UPDATE_TIME_TAB_TITLE_KEY = "updateTimeTabTitleKey"
+        const val UPDATE_DATE_TAB_TITLE_KEY = "updateDateTabTitleKey"
```

Comment:
Should these constants really be public?

Reply:
Its a key for all the fragments